src/app/view/components/_common/dot-icon-button/dot-icon-button.component.scss

```diff
@@ -1,30 +1,35 @@
 @import 'primeng-theme/utils/theme-variables';
+:host {
+    .dot-icon-button {
```

Comment:
You don't need a class here, even when it sound crazy, with the angular encapsulation you can do just: `button {}` because you only have one button in the component.